cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractUploaderTest.java

```diff
@@ -763,6 +763,13 @@ public void testAccessControl() throws ParseException, IOException {
         assertEquals("2019-03-21T22:00:00Z", accessControlResponse.get(0).get("end"));
     }
 
+    @Test
+    public void testOnSuccessScript() throws Exception {
+        String tags = "[\"autocaption\"" + ",\"" + SDK_TEST_TAG + "\",\"" + UPLOADER_TAG + "\"]";
+        Map result = cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("on_success", "current_asset.update({tags:" + tags + "});"));
+        assertEquals(result.get("tags"), asArray(new String[]{"autocaption"}));
```

Comment:
you are adding 3 tags but assert that they are equal 1. maybe change `to include`?

Reply:
Fix on its way